src/composables/multisigAccounts.ts

```diff
@@ -149,38 +148,44 @@ export function useMultisigAccounts({ store }: UseMultisigAccountsOptions) {
 
     const result = (await Promise.all(
       rawMultisigData.map(async ({ contractId, gaAccountId, ...otherMultisig }) => {
-        const contractInstance = await sdk.getContractInstance({
-          aci: SimpleGAMultiSigAci,
-          contractAddress: contractId,
-        });
-
-        const nonce = (await contractInstance.methods.get_nonce()).decodedResult;
-        const signers = (await contractInstance.methods.get_signers()).decodedResult;
-        const version = (await contractInstance.methods.get_version()).decodedResult;
-
-        const consensus = (
-          await contractInstance.methods.get_consensus_info()
-        ).decodedResult as any;
-
-        if (consensus.tx_hash) {
-          consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
+        try {
+          const contractInstance = await sdk.getContractInstance({
+            aci: SimpleGAMultiSigAci,
+            contractAddress: contractId,
+          });
+
+          const nonce = (await contractInstance.methods.get_nonce()).decodedResult;
+          const signers = (await contractInstance.methods.get_signers()).decodedResult;
+          const version = (await contractInstance.methods.get_version()).decodedResult;
+
+          const consensus = (
+            await contractInstance.methods.get_consensus_info()
+          ).decodedResult as any;
+
+          if (consensus.tx_hash) {
+            consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
+          }
+          consensus.expiration_height = Number(consensus.expiration_height);
+          consensus.confirmations_required = Number(consensus.confirmations_required);
+          consensus.totalConfirmations = Number(consensus.confirmed_by.length);
+
+          return {
+            ...camelcaseKeysDeep(consensus),
+            ...otherMultisig,
+            nonce: Number(nonce),
+            signers,
+            version,
+            contractId,
+            balance: (gaAccountId && (await sdk.balance(gaAccountId))) || 0,
```

Comment:
This await can be moved into separate `balance` constant and put together with in one `Promise.all` with `nonce`, `signers`, `version`. This is about avoiding too many consecutive awaits that probably makes this code slow.